diff --git a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx
index a0345b0..13ca0e6 100644
--- a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx
+++ b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx
@@ -142,8 +142,7 @@ const BottomSheetBackdropComponent = ({
 
   const AnimatedView = (
     <Animated.View
-      style={containerStyle}
-      pointerEvents={pointerEvents}
+      style={[containerStyle, pointerEvents ? { pointerEvents } : null]}
       accessible={_providedAccessible ?? undefined}
       accessibilityRole={_providedAccessibilityRole ?? undefined}
       accessibilityLabel={_providedAccessibilityLabel ?? undefined}
diff --git a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetBackground/BottomSheetBackground.tsx b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetBackground/BottomSheetBackground.tsx
index 11e7306..004eeba 100644
--- a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetBackground/BottomSheetBackground.tsx
+++ b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetBackground/BottomSheetBackground.tsx
@@ -8,11 +8,10 @@ const BottomSheetBackgroundComponent = ({
   style,
 }: BottomSheetBackgroundProps) => (
   <View
-    pointerEvents={pointerEvents}
     accessible={true}
     accessibilityRole="adjustable"
     accessibilityLabel="Bottom Sheet"
-    style={[styles.background, style]}
+    style={[styles.background, style, pointerEvents ? { pointerEvents } : null]}
   />
 );
 
diff --git a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetHostingContainer/BottomSheetHostingContainer.tsx b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetHostingContainer/BottomSheetHostingContainer.tsx
index 879061d..3714700 100644
--- a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetHostingContainer/BottomSheetHostingContainer.tsx
+++ b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetHostingContainer/BottomSheetHostingContainer.tsx
@@ -116,9 +116,8 @@ function BottomSheetHostingContainerComponent({
   return (
     <View
       ref={containerRef}
-      pointerEvents="box-none"
       onLayout={shouldCalculateHeight ? handleLayoutEvent : undefined}
-      style={containerStyle}
+      style={[containerStyle, { pointerEvents: 'box-none' }]}
       collapsable={true}
     >
       {children}
